pkg/model/gcemodel/api_loadbalancer.go

```diff
@@ -61,28 +61,32 @@ func createPublicLB(b *APILoadBalancerBuilder, c *fi.ModelBuilderContext) error
 	c.AddTask(poolHealthCheck)
 
 	ipAddress := &gcetasks.Address{
-		Name:      s(b.NameForIPAddress("api")),
-		Lifecycle: b.Lifecycle,
+		Name:         s(b.NameForIPAddress("api")),
+		ForAPIServer: true,
+		Lifecycle:    b.Lifecycle,
 	}
 	c.AddTask(ipAddress)
 
-	forwardingRule := &gcetasks.ForwardingRule{
+	c.AddTask(&gcetasks.ForwardingRule{
 		Name:       s(b.NameForForwardingRule("api")),
 		Lifecycle:  b.Lifecycle,
-		PortRange:  s("443-443"),
+		PortRange:  s(strconv.Itoa(wellknownports.KubeAPIServer) + "-" + strconv.Itoa(wellknownports.KubeAPIServer)),
 		TargetPool: targetPool,
 		IPAddress:  ipAddress,
 		IPProtocol: "TCP",
+	})
+	if b.Cluster.UsesNoneDNS() {
+		c.AddTask(&gcetasks.ForwardingRule{
```

Comment:
We may be able to lock this down e.g. only to the nodes... We/I can try that separately though!

Reply:
My idea is that, we could add another internal LB for this purpose, making it secure. Though, that would diverge from the original concept and a refactor would be needed to return the IP(s) for kops-controller separately from API ones.
My goal in this PR was to get it supported as simple as possible. To the very least, this gives us a chance to add some tests and see how the output changes later.